frontend/src/wordInteractions.js

```diff
@@ -48,7 +48,11 @@ function createNewRun () {
 function loadWordsFromApi () {
   fetch('http://localhost:3000/word_banks')
     .then(resp => resp.json())
-    .then(json => ALL_WORDS = json)
+    .then(words => {
+      for (const index in words) {
+        ALL_WORDS[words[index].id] = words[index].word;
```

Comment:
I would recommend using the word key, since we are treating it as a hash obj, whereas index usually refers to an integer/array.

Reply:
good point, will update